frontend/public/components/app-contents.tsx

```diff
@@ -380,6 +380,26 @@ const AppContents: React.FC<{}> = () => {
           )
         }
       />
+      <LazyRoute
```

Comment:
should we move the config map form components to dev-console package and use the dynamic extension for these routes

Reply:
`configmap` is a standard K8s object like secrets, routes etc. We should keep it in public (console/internal) package.

Reply:
I saw buildconfig form components are under dev-console package
@jerolimov ^